src/main/java/com/salesforce/dev/framework/DataDrivenManager.java

```diff
@@ -49,4 +51,92 @@ public DataDrivenManager(){
         return accountsArray.iterator();
     }
 
+    public Iterator<ViewSalesForce[]> getDataView(String fileJson){
+
+        Collection<ViewSalesForce[]> viewSalesForcesArray =new ArrayList<ViewSalesForce[]>();
+        try {
+            parser = new JSONParser();
+            String pathFileJson = "src\\test\\resources\\" + fileJson;
+            Object jsonObject = parser.parse(new FileReader(pathFileJson));
+            JSONArray jsonArray = (JSONArray) jsonObject;
+
+            ObjectMapper objectMapper = new ObjectMapper();
+
+            List<ViewSalesForce> navigation = objectMapper.readValue(jsonArray.toJSONString(),
+                    objectMapper.getTypeFactory().constructCollectionType(
+                            List.class, ViewSalesForce.class));
+
+            for (int i = 0 ; i< navigation.size(); i++){
```

Comment:
In this case is not necessary to use a for with index. It would be better to use a foreach
Review all your for sentences